conf/locale/ru/LC_MESSAGES/django.po

```diff
@@ -24723,4 +24727,4 @@ msgid ""
 msgstr ""
 "Если Вы не создавали учетную запись, дальнейших действий не требуется; "
 "Вы больше не будете получать от нас сообщения. Если Вам потребуется помощь, пожалуйста, не отвечайте на это сообщение. "
-"Обратитесь к разделу \"Помощь\" на веб-сайте {platform_name}."
+"Обратитесь к разделу \"Помощь\" на веб-сайте {platform_name}."
```

Comment:
please fix the no newline at the end of file

Reply:
I added the line